components/picker/stories/picker.stories.js

```diff
@@ -1,4 +1,6 @@
 import { Default as MenuStories } from "@spectrum-css/menu/stories/menu.stories.js";
+import { html } from "lit";
+import isChromatic from "chromatic/isChromatic";
```

Comment:
Can you update this with the new window.isChromatic() syntax now that that tool is merged?